packages/atomic/src/components/atomic-facet-manager/atomic-facet-manager.tsx

```diff
@@ -15,6 +15,10 @@ interface FacetElement extends HTMLElement {
   facetId: string;
 }
 
+/**
+ * The `atomic-facet-manager` reorders facets and their values to match the most recent search response with the most relevant results.
+ * A facet component must be defined within the opening and closing tags of an `atomic-facet-manager` to leverage this functionality.
```

Comment:
I think we can mention the "slot" concept here. Facets can be slotted inside a facet manager, meaning it can be inserted has a children. 
https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots#adding_flexibility_with_slots